indra/newview/llinventorylistener.cpp

```diff
@@ -61,78 +62,190 @@ LLInventoryListener::LLInventoryListener()
         &LLInventoryListener::getBasicFolderID,
         llsd::map("ft_name", LLSD(), "reply", LLSD()));
 
-    add("getDirectDescendents",
-        "Return the direct descendents(both items and folders) of the [\"folder_id\"]",
-        &LLInventoryListener::getDirectDescendents,
+    add("getDirectDescendants",
```

Comment:
I just copied the func name from llinventorymodel without thinking about spelling. 
Somehow most of our inventory-related functions have incorrect spelling (collectDescendEnts).

Reply:
I'm sorry, I did not mean to imply that the misspelling was your fault!

Reply:
No worries, I was just surprised that I hadn't noticed it before.